lib/compat/wordpress-6.2/block-patterns.php

```diff
@@ -9,54 +9,210 @@
  * Registers the block pattern categories REST API routes.
  */
 function gutenberg_register_core_block_patterns_and_categories() {
+	// Featured.
+	register_block_pattern_category(
+		'featured',
+		array(
+			'label' => _x( 'Featured', 'Block pattern category', 'gutenberg' ),
+			'description' => __( 'A set of high quality curated patterns.', 'gutenberg' ),
+		)
+	);
+
+	// Classic categories.
 	register_block_pattern_category(
 		'buttons',
 		array(
 			'label' => _x( 'Buttons', 'Block pattern category', 'gutenberg' ),
+			'description' => __( 'Patterns that contain buttons and call to actions.', 'gutenberg' ),
 		)
 	);
 	register_block_pattern_category(
 		'columns',
 		array(
 			'label' => _x( 'Columns', 'Block pattern category', 'gutenberg' ),
+			'description' => __( 'Multi column patterns with more complex layouts.', 'gutenberg' ),
 		)
 	);
 	register_block_pattern_category(
 		'footer',
 		array(
 			'label' => _x( 'Footers', 'Block pattern category', 'gutenberg' ),
+			'description' => __( 'A variety of footer designs displaying information and site navigation.', 'gutenberg' ),
 		)
 	);
 	register_block_pattern_category(
 		'gallery',
 		array(
 			'label' => _x( 'Gallery', 'Block pattern category', 'gutenberg' ),
+			'description' => __( 'Patterns containing mostly images or other media.', 'gutenberg' ),
 		)
 	);
 	register_block_pattern_category(
 		'header',
 		array(
 			'label' => _x( 'Headers', 'Block pattern category', 'gutenberg' ),
+			'description' => __( 'A variety of header designs displaying your site title and navigation.', 'gutenberg' ),
 		)
 	);
 	register_block_pattern_category(
 		'text',
 		array(
 			'label' => _x( 'Text', 'Block pattern category', 'gutenberg' ),
+			'description' => __( 'Patterns containing mostly text.', 'gutenberg' ),
 		)
 	);
 	register_block_pattern_category(
 		'query',
 		array(
 			'label'       => _x( 'Posts', 'Block pattern category', 'gutenberg' ),
-			'description' => __( 'Display post summaries in lists, grids, and other layouts.', 'gutenberg' ),
+			'description' => __( 'Display your latest posts in lists, grids or other layouts.', 'gutenberg' ),
 		)
 	);
+
+	/*
+	// Future categories, universal.
 	register_block_pattern_category(
-		'featured',
+		'call-to-action',
 		array(
-			'label' => _x( 'Featured', 'Block pattern category', 'gutenberg' ),
+			'label'       => _x( 'Call to Action', 'Block pattern category', 'gutenberg' ),
```

Comment:
These are all commented out for now, but I would expect them to replace all the existing categories.